doc/install_windows.txt

```diff
@@ -1,633 +1,177 @@
+.. include:: install_generic.inc
+    :start-line: 32
 
-.. note::
-
-    In Theano versions <= 0.5, ``theano-nose`` was not included.  If you
-    are working with such a version, you can call this command instead:
-
-    .. code-block:: bash
-
-        python theano/tests/run_tests_in_batch.py
-
-Compiling a faster BLAS
-~~~~~~~~~~~~~~~~~~~~~~~
-
-If you installed Python through WinPython or EPD, Theano will automatically
-link with the MKL library, so you should not need to compile your own BLAS.
-
-.. note::
-
-   The instructions below have not been tested in a Windows 64 bit environment.
-
-If you want a faster and/or multi-threaded BLAS library, you can
-compile OpenBLAS (ATLAS may work too, but was not tested, and is
-usually reported to be slower and more difficult to compile -- especially
-on Windows).
-OpenBLAS can be downloaded as a zip file from
-`its website <http://xianyi.github.io/OpenBLAS/>`__
-(we tested v0.2.6).
-To compile it, you will also need MSYS and wget (installation steps are
-described below).
-
-If you already have a full install of MinGW, you should
-have MSYS included in it, and thus should be able to start a MinGW shell.
-If that is the case, you can skip the following MSYS installation steps.
-Note that these steps were written for Python(x,y), but should also work
-for other bundle Python distributions like EPD (changing paths accordingly,
-for instance in EPD 7.3.2 the MinGW folder is
-``EPD7.3.2\EGG-INFO\mingw\usr\i686-w64-mingw32``).
-To install MSYS on top of the MinGW installation included within Python(x,y),
-do as follows:
-
-- Download the `mingw-get command-line installer binary
-  <http://sourceforge.net/projects/mingw/files/Installer/mingw-get/>`__.
-
-- Unpack its content into your ``pythonxy\mingw`` directory.
-
-- In a prompt (``cmd``), install MSYS with
+Configure CUDA and libgpuarray after installation
+=================================================
 
-    .. code-block:: bash
+Cuda
+----
 
-        mingw-get install msys-base
+To ensure Theano cand find CUDA headers and libraries, create an environment variable named ``CUDA_ROOT``
+and set it to the CUDA directory. In a 64-bits computer, the CUDA directory should be something like
+``C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v8.0``.
 
-  If ``mingw-get`` cannot be found automatically, just navigate first into the
-  folder were it was extracted (it is found in the ``bin`` subfolder).
+To create permanently an environment variable, look for ``system environment variables``
+in the Start search bar and then choose ``Set System Environment Variables`` (this works on Windows 8.1).
 
-- Edit ``pythonxy\mingw\msys\1.0\msys.bat`` (e.g. in Wordpad) and add as first
-  line ``set HOME=%USERPROFILE%``. Then create an easily accessible shortcut
-  (e.g. on your desktop) to this file, run it and within the MSYS
-  console, run the MSYS post-install script:
+libgpuarray
+-----------
 
-    .. code-block:: bash
+[TODO]
```

Comment:
You just have to make sure that the cuda dlls are in %PATH% (which they are by default) and libgpuarray will find them and load them.